### DISCUSSION TITLE
Autosearch - filesize limit

### DISCUSSION THREAD

**Discussion by Teh1Self** [airdcpp/airdcpp-windows#92] [Ideas]
Hi.

I would like to ask for feature, to set a filesize limitation to search.

For example, I am looking for a text file, but I dont want to download any size, just higher than 100KB, or between 100KB-1000KB

I apologize, if there is a way to do it, but I didn't figure it out.

Thanks in advance.

**Comment by Sopor** [comment]
I don't think it is possible to do for `Auto search` only for manual search.
